### PR TITLE
feat(settings): Make the AppScore component to use SVG icons

### PR DESCRIPTION
**Pull request by susnux**
## Summary
Use SVG icons instead of images for the app score component.

Screen Shot 2024-03-06 at 12 39 47
Screenshot 2024-03-06 at 12-38-06 Settings - Apps - Nextcloud

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [ ] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [ ] Screenshots before/after for front-end changes
- [x] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [x] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

**Review by szaimen — APPROVED** [review]
Code looks fine but didnt test

File: apps/settings/src/components/AppList/AppScore.vue [hunk 1]
```diff
@@ -21,22 +22,70 @@
...
+		hasHalfStar() {
+			return (this.fullStars + this.emptyStars) < 5
+		},
+	},
+})
 </script>
+<style scoped>
+.app-score__wrapper {
+	display: inline-flex;
+	color: #a08b00;
```
Comment (szaimen) [inline apps/settings/src/components/AppList/AppScore.vue:84]: can we use the variable from https://github.com/nextcloud/server/pull/44034 here?

Comment (susnux) [reply apps/settings/src/components/AppList/AppScore.vue:84]: yes done

**Review by szaimen — APPROVED** [review]

File: apps/settings/src/components/AppList/AppScore.vue [hunk 2]
```diff
@@ -21,22 +22,69 @@
...
+		hasHalfStar() {
+			return (this.fullStars + this.emptyStars) < 5
+		},
+	},
+})
 </script>
+<style scoped>
+.app-score__wrapper {
+	display: inline-flex;
+	color: var(--color-favorite, #a08b00);
```
Comment (Pytal) [inline apps/settings/src/components/AppList/AppScore.vue:84]:
```suggestion
	color: var(--color-favorite, #a08b00);
```
Will be added as a new variable in followup?

**Review by Pytal — APPROVED** [review]

**Pull request merged**